main.cpp

```diff
@@ -52,84 +53,110 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bicycle
+//  action 1: the bicycle rides
+bicycle.ride();
+//  action 2: the bicycle flies
+bicycle.fly();
+//  action 3: the bicycle dings its bell
+bicycle.bells();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: lavalamp
+//  action 1: lavalamp is turned ON
+lavalamp.on();
+//  action 2: lavalamp rotates
+lavalamp.rotate();
 
-//  action 3:
+//  action 3: lavalamp plays music
+lavalamp.music();

```

Comment:
the action is `plays`
the pseudo code needs to say this.
`music` is not an action, it is a noun.
- [ ] revise